indico/modules/events/registration/forms.py

```diff
@@ -60,6 +61,7 @@ class RegistrationFormForm(IndicoForm):
                               filters=[lambda x: x if x is not None else 0],
                               widget=NumberInput(step='0.01'),
                               description=_("A fixed fee all users have to pay when registering."))
+    notification_sender_address = StringField(_('Notification sender address'), [IndicoEmail()])
```

Comment:
I think this will store an empty string instead of NULL when saving the form, which might result in an empty sender  address. You can avoid this using `filters=[lambda x: (x or None)]` on the field